sparrow-blog-main/src/main/java/com/sparrow/blog/controller/PostsController.java

```diff
@@ -0,0 +1,45 @@
+package com.sparrow.blog.controller;
+
+
+import com.sparrow.blog.controller.base.BaseController;
+import com.sparrow.blog.protocol.dto.PostsSearchRequestDTO;
+import com.sparrow.blog.protocol.dto.PostsSearchResponseDTO;
+import com.sparrow.blog.service.PostsSevice;
+import com.sparrow.protocol.Result;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("posts")
+public class PostsController extends BaseController {
+    @Autowired
+    private PostsSevice postsSevice;
```

Comment:
Please use the constructor to inject the `bean`
```suggestion
     private PostsSevice postsSevice;
    public PostsController(PostsSevice postsSevice) {
        this.postsSevice = postsService;
    }
```

Reply:
officely just recommend it ，not mandatory。

Reply:
Yeah, I know it, but I still recommended remove the project warning code